src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -1878,10 +1880,81 @@ protected override void ProcessRecord()
             }
         }
 
+        internal bool IsModuleInDenyList(string[] moduleDenyList, string moduleName, ModuleSpecification moduleSpec)
```

Comment:
'private' accessibility for helper method?

Reply:
updated.